client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -133,28 +133,39 @@ class PostRenderBallotProcessor extends Handler {
     if (pages.length % 2) {
       const pagedjsPages = pages[0].pagesArea
       if (pagedjsPages.lastChild) {
-        pagedjsPages.appendChild(pagedjsPages.lastChild.cloneNode(true))
+        const newLastPageElement = pagedjsPages.lastChild.cloneNode(
+          true
+        ) as HTMLElement
+        pagedjsPages.appendChild(newLastPageElement)
         pagedjsPages.setAttribute(
           'style',
           `--pagedjs-page-count:${pages.length + 1};`
         )
-        const lastPage = pagedjsPages.lastChild! as Element
-        lastPage.id = `page-${pages.length + 1}`
-        lastPage.classList.remove('pagedjs_first_page', 'pagedjs_right_page')
-        lastPage.classList.add('pagedjs_left_page')
-        lastPage.setAttribute('data-page-number', `${pages.length + 1}`)
+        newLastPageElement.id = `page-${pages.length + 1}`
+        newLastPageElement.classList.remove(
+          'pagedjs_first_page',
+          'pagedjs_right_page'
+        )
+        newLastPageElement.classList.add('pagedjs_left_page')
+        newLastPageElement.setAttribute(
+          'data-page-number',
+          `${pages.length + 1}`
+        )
+        newLastPageElement.setAttribute('data-id', `page-${pages.length + 1}`)
```

Comment:
Is there a reason not to use the approach we had, i.e.

```suggestion
        newLastPageElement.dataset.pageNumber = `${pages.length + 1}`
        newLastPageElement.dataset.id = `page-${pages.length + 1}`
```

Reply:
maybe, but the approach we had was via `setAttribute` so I didn't want to change it. I added the `id` one cause it was missing.